var/spack/repos/builtin/packages/flecsi/package.py

```diff
@@ -61,4 +64,10 @@ def build_type(self):
             return 'Release'
 
     def cmake_args(self):
-        return ['-DENABLE_UNIT_TESTS=ON']
+        options = ['-DENABLE_UNIT_TESTS=ON']
```

Comment:
Does this flag cause the unit tests to run? If so, you should make it optional:
```python
options = []

if self.run_tests:
    options.append('-DENABLE_UNIT_TESTS=ON')
```
Then the tests will be run only when a user explicitly asks for them:
```
$ spack install --run-tests
```